### PR TITLE
Move announcements functionality to a separate Django app...

### PR DESCRIPTION
**Pull request by ptormene**
...and add it to INSTALLED_APPS only if the webui is set to require authentication.

NOTE: in machines where we already created the `announcements` table, it will remain orphan unless we rollback the corresponding old migration and delete the table. I would do the rollback for instance in the aelo-beta machine.

### CODE REVIEW

File: openquake/server/settings.py [hunk 1]
```diff
@@ -305,6 +305,7 @@
         'django.contrib.messages',
         'django.contrib.sessions',
         'django.contrib.admin',
+        'announcements',
```
Comment (micheles) [inline openquake/server/settings.py:308]: I was expecting a name like `openquake.server.announcements`

Comment (ptormene) [reply openquake/server/settings.py]: Fixed, thanks

**Comment by micheles** [comment]
Do we have a test reading the announcement table in the non-public mode?

**Review by micheles — APPROVED** [review]

**Comment by micheles** [comment]
Please update the changelog since the announcements are a new feature

**Pull request merged**